src/services/__mocks__/groupsService.ts

```diff
@@ -177,3 +177,18 @@ export const deletePartyGroup = (
   );
   return new Promise<void>((resolve) => resolve());
 };
+
+export const deleteGroupRelation = (
+  _party: Party,
+  _product: Product,
+  group: PartyGroupExt,
+  _userId: string
+): Promise<any> => {
+  const selectedGroup = mockedGroups.find((g) => g.id === group.id);
+  // eslint-disable-next-line functional/immutable-data
+  selectedGroup?.membersIds.splice(
+    selectedGroup?.membersIds.findIndex((x) => x),
```

Comment:
there is an error on this line, in this way you will find the first element, even if it has not the same userId.
Use indexOf instead of findIndex in order to search for the index of the _userId variable